packages/web-app-files/src/views/PublicFiles.vue

```diff
@@ -176,7 +178,31 @@ export default defineComponent({
 
   methods: {
     ...mapActions('Files', ['loadPreview']),
-    ...mapMutations('Files', ['SET_CURRENT_FOLDER', 'LOAD_FILES', 'CLEAR_CURRENT_FILES_LIST']),
+    ...mapMutations('Files', ['SET_CURRENT_FOLDER', 'LOAD_FILES', 'CLEAR_CURRENT_FILES_LIST', 'REMOVE_FILE', 'REMOVE_FILE_FROM_SEARCHED', 'REMOVE_FILE_SELECTION']),
+
+    async fileDropped(fileIdTarget) {
+      const selected = [...this.selectedResources]
+      const targetInfo = this.paginatedResources.find((e) => e.id === fileIdTarget)
+      const isTargetSelected = selected.some((e) => e.id === fileIdTarget)
+      if (isTargetSelected) return
+      if (targetInfo.type !== 'folder') return
+      const movedResources = await move(
```

Comment:
the `PublicFiles` view will need different handling inside the `copyMove.ts` file regarding SDK calls (`client.publicFiles` instead of `client.files`)